### PR TITLE
feat: separate mainnet by network

### PR DESCRIPTION
**Pull request by pkucherenk0**

### CODE REVIEW

File: networks/polygon/markets.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+    "spot://DUCKIES/USDC",
+    "spot://USDT/USDC",
+    "spot://MATIC/USDC",
+    "spot://WBTC/USDC",
+    "spot://WETH/USDC",
+    "spot://SHIBA/USDC",
+    "spot://LINDA/USDC"
```
Comment (mnaichuk) [inline networks/polygon/markets.json:8]: I'm not sure they have linda on polygon

Comment (mnaichuk) [reply networks/polygon/markets.json:8]: and as we don't have it in assets json

File: networks/polygonMumbai/markets.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[
+    "spot://DUCKIES/USDC",
+    "spot://USDT/USDC",
+    "spot://MATIC/USDC",
+    "spot://WBTC/USDC",
+    "spot://WETH/USDC",
+    "spot://SHIBA/USDC",
+    "spot://LINDA/USDC"
```
Comment (mnaichuk) [inline networks/polygonMumbai/markets.json:8]: same here

**Comment by mod** [comment]
over looks okay but use the official network name we know mumbai is a testnet

**Pull request merged**